cmd/bridge/main.go

```diff
@@ -207,18 +203,18 @@ func main() {
 
 	if *fCustomLogoFile != "" {
 		if _, err := os.Stat(*fCustomLogoFile); err != nil {
-			log.Fatalf("could not read logo file: %v", err)
+			klog.Fatalf("could not read logo file: %v", err)
 		}
 	}
 
 	if *fInactivityTimeout < 300 {
-		log.Warning("Flag inactivity-timeout is set to less then 300 seconds and will be ignored!")
+		klog.Warning("Flag inactivity-timeout is set to less then 300 seconds and will be ignored!")
```

Comment:
@zherman0  For changes from `log` to `klog`, when viewing console logs, how can we differentiate them? 
On a 4.6 cluster, I see console log is showing:
```
2020-10-19T23:47:06Z cmd/main: Flag inactivity-timeout is set to less then 300 seconds and will be ignored!
```

While on a cluster with changes in this PR, it shows:
```
sh-4.4$ /opt/bridge/bin/bridge --log-level 5
W1020 05:55:37.303295      25 main.go:211] Flag inactivity-timeout is set to less then 300 seconds and will be ignored!
W1020 05:55:37.303578      25 main.go:254] DEPRECATED: --log-level is now deprecated, use verbosity flag --v=Level instead
```
How can I know `Flag inactivity-timeout is set to less then 300 seconds and will be ignored!` is printed with `klog`?

Reply:
@yapei not sure why we need to differentiate between them? 
> How can I know Flag inactivity-timeout is set to less then 300 seconds and will be ignored! is printed with klog?

Since we will be using just `klog` for the backend logging we dont need to differentiate between anything.

Reply:
Agreed, not sure why you need to tell the difference, but the new klog stuff shows the file name and line number `main.go:211` if that helps.

Reply:
Yeah, the line number tells the truth @zherman0 
When verifying the bug `Logging is broken due to mix of k8s.io/klog v1 and v2` we need to confirm whether klog v2 is used.  So I thought for this PR we may have same verification process. @jhadvig  @zherman0

Reply:
@yapei @spadgett @zherman0 just one thing to notice here. The `console` banckend will using the `klog/v1` since we need to update the `library-go` if we want to use the `klog/v2`. On the other hand the `console-operator` moved by [this PR](https://github.com/openshift/console-operator/commit/34999dc4bbe85e37f40f4fc426a9ad5091666695) to using the `klog/v2`. This should not do any hard since those are two different binaries.
Just want to point out the difference here and also think out loud about the upgrading to `klog/v2` for the console backend binary.